q2_diversity_lib/tests/test_alpha.py

```diff
@@ -234,3 +239,42 @@ def test_do_not_drop_undefined_samples(self):
         actual = shannon_entropy(table=self.input_table,
                                  drop_undefined_samples=False)
         pdt.assert_series_equal(actual, self.expected)
+
+
+class AlphaPassthroughTests(TestPluginBase):
+    package = 'q2_diversity_lib.tests'
+
+    def setUp(self):
+        super().setUp()
+        self.method = self.plugin.actions['alpha_passthrough']
+        self.available_metrics = alpha.METRICS['NONPHYLO']['UNIMPL']
+        empty_table = biom.Table(np.array([]), [], [])
+        self.empty_table = Artifact.import_data('FeatureTable[Frequency]',
+                                                empty_table)
+        crawford_tbl = self.get_data_path('crawford.biom')
+        self.crawford_tbl = Artifact.import_data('FeatureTable[Frequency]',
+                                                 crawford_tbl)
+
+    def testMethod(self):
```

Comment:
Should this be `test_method`?

```suggestion
    def test_method(self):
```